ansible/roles/ocp4-workload-pam-fraudmanagement-workshop-verification/tasks/workload_per_project_businessautomationworkload.yml

```diff
@@ -1,11 +1,16 @@
 ---
 - name: Check {{ _case_kie_container_id }} responds with 201
   uri:
-    url: "https://{{ kieserver_host.stdout }}/services/rest/server/containers/{{ _case_kie_container_id }}/cases/proactivefrauddetectioncase.fraud-case/instances"
+    url: >-
```

Comment:
This should be made more readable. Either disable yamllint here or use something like this. Or both.

```
url: >-
  {{ 'https://{0}/services/rest/server/containers/{1}/cases/proactivefrauddetectioncase.fraud-case/instances'.format(
  kieserver_host.stdout, _case_kie_container_id) }}
```

Reply:
Fixed it .. because the url exceeded 120 chars, I placed the `{{` and the url on different lines (yamllint will ignore long lines that don't have spaces in them).

Reply:
looks good.